src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicWithHorizon.hpp

```diff
@@ -73,12 +73,13 @@ struct EvolutionMetavars<3, InitialData, BoundaryConditions>
 
   struct AhA : tt::ConformsTo<intrp::protocols::InterpolationTargetTag> {
     using temporal_id = ::Tags::Time;
-    using tags_to_observe = ::ah::tags_for_observing;
+    using tags_to_observe = ::ah::tags_for_observing<Frame::Inertial>;
```

Comment:
Wouldn't the horizon quantities in this executable also be wrong in the inertial frame? If it's an issue with the maps?

Reply:
Nope. I think it's an issue with the map but only when the frames are different